Source/BoardGameGeekApiClient/Service/ApiDownloaderService.cs

```diff
@@ -26,6 +27,7 @@ public class ApiDownloaderService : IApiDownloadService
 
         public ApiDownloaderService()
         {
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
```

Comment:
I guess BGG upgraded their TLS to version 1.2 and it was throwing an exception before it even hits your code that logs the status code.